src/baldr/attributes_controller.cc

```diff
@@ -1,9 +1,10 @@
-#include <thor/attributes_controller.h>
+#include "baldr/attributes_controller.h"
+#include "midgard/logging.h"
 
 #include <string>
 
 namespace valhalla {
-namespace thor {
+namespace baldr {
```

Comment:
Whats the reason for moving this to `baldr`?

Reply:
this was so that we could re-use this class in odin when we serialize the route out. im not sure its the best place, we could hvae argued to put it in tyr also i think. if you like tyr better we could totally move that in the next pr. also sorry for not answering this question earlier somehow i didnt see it!